custom/icds_reports/migrations/sql_templates/database_views/thr_report_view.sql

```diff
@@ -25,8 +25,8 @@ SELECT
 COALESCE(agg_awc.thr_distribution_image_count,0) as thr_distribution_image_count,
 agg_awc.is_launched,
 agg_awc.month as month,
-COALESCE(SUM(agg_child_health.rations_21_plus_distributed),0) + COALESCE(agg_awc.num_mother_thr_21_days,0) as thr_given_21_days,
-COALESCE(SUM(agg_child_health.thr_eligible),0) + COALESCE (agg_awc.num_mother_thr_eligible,0) as total_thr_candidates
+COALESCE(SUM(agg_awc.thr_rations_21_plus_distributed_child),0) + COALESCE(agg_awc.num_mother_thr_21_days,0) as thr_given_21_days,
```

Comment:
It is not making sense to me to make different to deploy this view change and wait for 1 week this change to reflect.
for this, I can do that data updation in the agg table on the night of deploy before people wakes up next day